### DISCUSSION TITLE
Langfuse SDK with Spring AI

### DISCUSSION THREAD

**Discussion by bntv-config** [langfuse/langfuse#4599] [Ideas]
### Describe the feature or potential improvement

Please release SDK that we can use with Spring AI

**Comment by marcklingen** [comment]
Thanks for sharing, do you mean this Spring AI? https://spring.io/projects/spring-ai

**Reply by bntv-config** [reply]
Yes exactly. I want to use the tracing and finops capability with Spring AI using langfuse

**Reply by marcklingen** [reply]
ok eprfect, currently this is not available but you could create your own Langfuse client based on the openapi spec: https://cloud.langfuse.com/generated/api/openapi.yml

**Comment by marcklingen** [comment]
Langfuse can now act as an OTel backend. Please let me know in case this doesn’t work well with Spring!

Langfuse docs: https://langfuse.com/docs/opentelemetry/get-started
Spring docs: https://docs.spring.io/spring-ai/reference/observability/index.html
